Lib/fontTools/feaLib/ast.py

```diff
@@ -477,6 +548,11 @@ def asFea(self, indent=""):
 
 
 class Anchor(Expression):
+    """An ``Anchor`` element, used inside a ``pos`` rule.
+
+    If a ``name`` is given, this will be used in preference to the coordinates.
+    Other values should be integer.
```

Comment:
“integer” should be “integers”. Also, the other values can be `None` instead of integers, and the device tables should not be integers.